kuma/landing/views.py

```diff
@@ -51,17 +51,82 @@ def promote_buttons(request):
     return render(request, 'landing/promote_buttons.html')
 
 
+ROBOTS_ALLOWED_TXT = '''\
```

Comment:
I think it can be in a jinja template so that it can be generated in future

Reply:
What's an example of something that would be templatized?

Reply:
Like the `Sitemap`, it should be dependent upon host
`Sitemap: https://developer.mozilla.org/sitemap.xml` should be `settings.HOST + /sitemap.xml`

Reply:
It's an error if we're telling search engines that the sitemap for indexed content is anywhere other than https://developer.mozilla.org/sitemap.xml. It seems like we're asking for trouble if we allow it to be configured by settings.